api/v1alpha1/flowcollector_types.go

```diff
@@ -219,19 +222,63 @@ type FlowCollectorEBPF struct {
 type FlowCollectorKafka struct {
 	// Important: Run "make generate" to regenerate code after modifying this file
 
+	// InstanceSpec is the Spec section of LokiStack instance.
```

Comment:
To follow the API guidelines:
```suggestion
	// instanceSpec is the Spec section of LokiStack instance.
```